crates/pg_schema_cache/src/queries/columns.sql

```diff
@@ -0,0 +1,60 @@
+with
+  available_tables as (
+    select
+      c.relname as table_name,
+      c.oid as table_oid,
+      c.relkind as class_kind,
+      n.nspname as schema_name
+    from
+      pg_catalog.pg_class c
+      join pg_catalog.pg_namespace n on n.oid = c.relnamespace
+    where
+      -- r: normal tables
+      -- v: views
+      -- m: materialized views
+      -- f: foreign tables
+      -- p: partitioned tables
+      c.relkind in ('r', 'v', 'm', 'f', 'p')
+  ),
+  available_indexes as (
+    select
+      unnest (ix.indkey) as attnum,
+      ix.indisprimary as is_primary,
```

Comment:
Note that if we're using compounds keys, e.g. `(user_id, tax_number)`, we'd have `is_primary == true` for `tax_number`. 

This might be problematic for index suggestions etc. – wdyt?